### PR TITLE
DYN-4735-ContextMenu-FixedWidth

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Fixed width for ContextMenus "Group Styles", "Colors" and "Font Size"
I've added a new style that will set a Fixed Width for the Sub ContextMenus in "Colors" and "Font Size" entries, the "Group Style" entry the style was also modified for fit the expected Width.

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Fixed width for ContextMenus "Group Styles", "Colors" and "Font Size"

### Reviewers

@QilongTang

### CODE REVIEW

**Comment by RobertGlobant20** [comment]
This is a GIF showing the look and feel of the context menus.
group_style_fixed_contextmenu

File: src/DynamoCoreWpf/Views/Core/AnnotationView.xaml [hunk 1]
```diff
@@ -383,7 +385,9 @@
                         </ListBox.Items>
                     </ListBox>
                 </MenuItem>
-                <MenuItem Name="ChangeSize" Header="{x:Static p:Resources.GroupContextMenuFont}">
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Core/AnnotationView.xaml:386]: Should we remove either of these if no longer used?

Comment (RobertGlobant20) [reply src/DynamoCoreWpf/Views/Core/AnnotationView.xaml:386]: @QilongTang the ChangeSize menuitem is still used my change was only applying a Style to the MenuItem so we can have a fixed Width.

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
The PR validation reported no test failure but the check did not pass. Merging

**Comment by QilongTang** [comment]
@RobertGlobant20 Would you make sure this branch is up to date with master without the timeout problem?

**Comment by QilongTang** [comment]
Passed at https://master-15.jenkins.autodesk.com/view/DYN/job/DYN-DevCI_Self_Service/952/

**Pull request merged**